### PR TITLE
Document correction

### PR DESCRIPTION
**Pull request by naveenkandakur**

### CODE REVIEW

File: plugins/modules/powervm_dlpar.py [hunk 1]
```diff
@@ -19,8 +19,9 @@
     - Navinakumar Kandakur(@nkandak1)
 short_description: Dynamically managing resources of partition
 notes:
-    - In case of update_pv action if updation of settings belongs to atleast one VIOS is successful then
-      changed status shown as 1 and failed updates are shown as warning, This behavior is same for update_npiv and update_vod also.
+    - In case of update_pv action,  If updating of settings belongs to at least one VIOS is successful then
```
Comment (AnilVijayan) [inline plugins/modules/powervm_dlpar.py]: If the updating of settings for at least one VIOS is successful during the execution of the "update_pv" action, the changed status will be displayed as 1, and any failed updates will be shown as warnings. This behavior remains consistent for both the "update_npiv" and "update_vod" actions.

File: plugins/modules/powervm_dlpar.py [hunk 2]
```diff
@@ -19,8 +19,9 @@
...
+      changed status shown as 1 and failed updates are shown as warning, This behaviour is same for update_npiv and update_vod also.
+    - None of the operations support passwordless authentication.
```
Comment (AnilVijayan) [inline plugins/modules/powervm_dlpar.py]: Passwordless authentication is not supported.

File: plugins/modules/vios.py [hunk 3]
```diff
@@ -187,6 +189,17 @@
     name: <vios_partition_name>
     action: accept_license
 
+- name: Shaow VIOS details with Free PVs and Virtual Optical Media.
```
Comment (AnilVijayan) [inline plugins/modules/vios.py]: typo

**Review by AnilVijayan — CHANGES_REQUESTED** [review]

File: plugins/modules/vios.py (hunk 3 above)
Comment (naveenkandakur) [reply plugins/modules/vios.py]: Fixed

File: plugins/modules/powervm_dlpar.py (hunk 2 above)
Comment (naveenkandakur) [reply plugins/modules/powervm_dlpar.py]: Fixed

File: plugins/modules/powervm_dlpar.py (hunk 1 above)
Comment (naveenkandakur) [reply plugins/modules/powervm_dlpar.py]: Fixed

**Pull request merged**